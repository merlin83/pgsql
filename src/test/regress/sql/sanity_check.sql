--
-- sanity check, if we don't have indices the test will take years to
-- complete.
--
SELECT relname, relhasindex
   FROM pg_class
   WHERE relhasindex
   ORDER BY relname;

