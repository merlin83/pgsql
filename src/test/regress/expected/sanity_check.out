QUERY: VACUUM;
QUERY: SELECT relname, relhasindex
   FROM pg_class
   WHERE relhasindex
   ORDER BY relname;
relname       |relhasindex
--------------+-----------
bt_f8_heap    |t          
bt_i4_heap    |t          
bt_name_heap  |t          
bt_txt_heap   |t          
fast_emp4000  |t          
hash_f8_heap  |t          
hash_i4_heap  |t          
hash_name_heap|t          
hash_txt_heap |t          
ihighway      |t          
onek          |t          
pg_attrdef    |t          
pg_attribute  |t          
pg_class      |t          
pg_description|t          
pg_proc       |t          
pg_relcheck   |t          
pg_trigger    |t          
pg_type       |t          
road          |t          
shighway      |t          
tenk1         |t          
tenk2         |t          
(23 rows)

